R/classicProcess.R

```diff
@@ -1926,6 +1928,54 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   }
 }
 
+
+.procRsquared <- function(options, modelsContainer) {
+  # adapted from .semRsquared() from jaspSem/R/sem.R (one table for several models)
+  if (!options[["rSquared"]] || !is.null(modelsContainer[["rsquared"]])) return()
+  
+  # retrieve model objects
+  procResults <- lapply(options[["processModels"]], function(mod) modelsContainer[[mod[["name"]]]][["fittedModel"]]$object)
+  modelNames <- sapply(options[["processModels"]], function(mod) mod[["name"]])
+  
+  # init table
+  tabr2 <- createJaspTable(gettext("R-Squared"))
```

Comment:
```suggestion
  tabr2 <- createJaspTable(gettext("R-squared"))
```

For consistency with the option name in the GUI and the capitalization with the other output.